modules/standard/Random.chpl

```diff
@@ -754,23 +754,23 @@ module Random {
     record SeedGenerator {
       /*
         Generate a seed based on the current time in microseconds as
-        reported by :proc:`Time.getCurrentTime`. This seed is not
+        reported by :proc:`Time.timeSinceEpoch`. This seed is not
         suitable for the NPB RNG since that requires an odd seed.
       */
       proc type currentTime: int(64) {
         use Time;
-        const seed = getCurrentTime(unit=TimeUnits.microseconds):int(64);
+        const seed = (timeSinceEpoch().totalSeconds()*1_000_000):int(64);
```

Comment:
The last 6 digits won't actually be zero because `totalSeconds()` returns a real that includes the microseconds part after the decimal point. The main worry I had about this change was that it could lose precision due to the seconds since the Unix epoch being significantly larger than the seconds since midnight.

Reply:
Ok, my bad, I didn't realize it was a `real`. I think this change makes sense then. Thanks!

Reply:
Cool, thank you!